source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm

```diff
@@ -262,7 +291,7 @@ - (NSString *)getTitlesOfChoices
 
 - (float)getNonInputWidth:(UITableViewCell *)cell
 {
-    return cell.separatorInset.left + cell.indentationWidth + cell.accessoryView.frame.size.width + cell.imageView.image.size.width;
+    return padding * 3 + cell.imageView.image.size.width;
```

Comment:
> padding * 3 +  [](start = 10, length = 15)

What is the magic number here?

Reply:
it's the padding that's put before the cell's image, the front of the row, and the end of the row.